### PR TITLE
How to install SinusBot on Linux [DE/EN]

### PR DESCRIPTION
**Pull request by HerrDerMelonen**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: c.akoutas@live.de

### CODE REVIEW

File: tutorials/install-sinusbot/01.de.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+#### Aber was ist SinusBot eigentlich?
+
+SinusBot ist ein Programm, das es dir ermöglicht deine Lieblings Musik zusammen mit deinen Freunden auf Teamspeak zu hören.
+
+**Voraussetzung**
+
++ Ein 64-Bit kompatibles OS
++ Ubuntu 14.04+/ Debian 8+
++ Ein CX21 Hetzner CloudServer oder besser
```
Comment (n-se) [inline tutorials/install-sinusbot/01.de.md]: CX21? I think Sinusbot also runs good on a CX11

File: tutorials/install-sinusbot/01.de.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+Bevor wir loslegen sollte dein CloudServer aktualisiert werden.
+```bash
+apt-get update && apt-get -y upgrade
+```
+SinusBot benötigt verschiedene Abhängigkeiten die dringend Installiert werden müssen!
+```bash
+apt-get install -y x11vnc xvfb libxcursor1 ca-certificates bzip2 libnss3 libegl1-mesa x11-xkb-utils libasound2 libglib2.0-0 libgl1 libnspr4 libfontconfig1 libxi6 libxcursor1 libxcomposite1 libasound2 libxtst6
+```
+Aktualisieren der Zertifikate durch ```ca-certificates```.
```
Comment (n-se) [inline tutorials/install-sinusbot/01.de.md]: For single words or one-liner, better only use a single ` at beginning and end. Please replace this in the tutorial.

File: tutorials/install-sinusbot/01.de.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+SinusBot ist ein Programm, das es dir ermöglicht deine Lieblings Musik zusammen mit deinen Freunden auf Teamspeak zu hören.
+
+**Voraussetzung**
+
++ Ein 64-Bit kompatibles OS
++ Ubuntu 14.04+/ Debian 8+
++ Ein CX21 Hetzner CloudServer oder besser
++ Teamspeak
+
+## Step 1 - Vorbereitungen <summary of step>
```
Comment (n-se) [inline tutorials/install-sinusbot/01.de.md]: You can remove <summary of step>. I noticed you have left this here often in the tutorial, please remove it.

File: tutorials/install-sinusbot/01.de.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+```
+Damit SinusBot funktioniert muss im Installationsverzeichnis ein Teamspeak Client installiert und die Rechte angepasst werden.
+```bash
+wget http://dl.4players.de/ts/releases/3.2.3/TeamSpeak3-Client-linux_amd64-3.2.3.run && chmod 0755 TeamSpeak3-Client-linux_amd64-3.2.3.run
+```
+Jetzt muss der Teamspeak Client Entpackt werden.
+```bash
+./TeamSpeak3-Client-linux_amd64-3.2.3.run
+```
++ #### Im Entpackprozess nacheinander ```Enter``` + ```q``` + ```y``` + ```Enter``` drücken!
```
Comment (n-se) [inline tutorials/install-sinusbot/01.de.md]: Is it expected to have a `+` here?

File: tutorials/install-sinusbot/01.de.md (hunk 1 above)
Comment (HerrDerMelonen) [reply tutorials/install-sinusbot/01.de.md]: Hab es zu CX11 geändert.

File: tutorials/install-sinusbot/01.de.md (hunk 2 above)
Comment (HerrDerMelonen) [reply tutorials/install-sinusbot/01.de.md]: Hab das ebenfalls mit zu einem ` geändert.

File: tutorials/install-sinusbot/01.de.md (hunk 3 above)
Comment (HerrDerMelonen) [reply tutorials/install-sinusbot/01.de.md]: Hab entfernt.

File: tutorials/install-sinusbot/01.de.md (hunk 4 above)
Comment (HerrDerMelonen) [reply tutorials/install-sinusbot/01.de.md]: Ist so vorgesehen da ich dies eingerückt und mit einem Punkt haben wollte.

**Pull request merged**